### PR TITLE
For 4.0.15:  tikv(cdc): dynamic online change cdc configuration

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add dynamic online change cdc configuration docs.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @amyangfei @overvenus 

Could you please take a look? Thanks!

**Review by overvenus — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@hi-rustin Only v4.0 needs this content?

**Comment by hi-rustin** [comment]
> @hi-rustin Only v4.0 needs this content?

needed, but the configuration is different. I'll commit a master PR and then cherry-pick to the other branches. I'm committing 4.0 separately because it's about to be released.

**Comment by TomShawn** [comment]
> > @hi-rustin Only v4.0 needs this content?
> 
> needed, but the configuration is different. I'll commit a master PR and then cherry-pick to the other branches. I'm committing 4.0 separately because it's about to be released.

Is it about to be released in v4.0.15?

**Comment by hi-rustin** [comment]
> 
> Is it about to be released in v4.0.15?

yes

**Review by TomShawn — APPROVED** [review]
LGTM

File: dynamic-config.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ show warnings;
 | split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果读 QPS 连续 10 秒内均超过这个值，则进行 split |
 | split.split-balance-score | load-base-split 的控制参数，确保 split 后左右访问尽量均匀 |
 | split.split-contained-score | load-base-split 的控制参数，尽量减少 split 后跨 Region 访问 |
+| cdc.incremental-scan-concurrency | 增量扫描历史数据任务的最大并发执行个数 |
+| cdc.incremental-scan-speed-limit| 增量扫描历史数据的速度上限 |
+| cdc.min-ts-interval | 定期推进 Resolved TS 的时间间隔 |
+| cdc.sink-memory-quota| 缓存在内存中的 TiCDC 数据变更事件占用内存的上限 |
```
Comment (TomShawn) [inline dynamic-config.md:202]:
```suggestion
| cdc.incremental-scan-concurrency | 增量扫描历史数据任务的最大并发执行个数（自 v4.0.15 起支持） |
| cdc.incremental-scan-speed-limit| 增量扫描历史数据的速度上限（自 v4.0.15 起支持） |
| cdc.min-ts-interval | 定期推进 Resolved TS 的时间间隔（自 v4.0.15 起支持）  |
| cdc.sink-memory-quota| 缓存在内存中的 TiCDC 数据变更事件占用内存的上限（自 v4.0.15 起支持）  |
```

**Pull request merged**